### DISCUSSION TITLE
Can't get my Chia to be vissible in the wallet

### DISCUSSION THREAD

**Discussion by xxxxxSly** [Chia-Network/chia-blockchain#18740] [Support]
Hi, I have an Issue. On my harvester there is some amount of Chia, but I can't see them in my wallet. What could be the reason and solution? It's working 3 days non stop, so there is no closing issue.

**Comment by BrandtH22** [comment]
hey @xxxxxSly , the previous poster is a spammer do not click their link and if you have please cease any contact with them immediately. 

Can you provide more information on the question? By on the harvester do you have two separate wallets or is it that you have farmed a block solo and need to claim the funds?

Have you tried a wallet resync with the gui settings / advanced / resync option? (this will close the chia client and upon reopening will resync the wallet)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
